### ISSUE TITLE
question about the split hotpot dataset?

### ISSUE THREAD

**Issue by Aleczhang13** [PrekshaNema25/RefNet-QG#8]
Hi! I have read your paper and I have some related questions about split the dataset to ask. In the paper, you said you use 10% of the training data for validation and use the original dev set as the test set for the hotpot dataset. So I wonder to know the details of how to split the hotpot dataset.

**Comment by Aleczhang13** [comment]
1. Do you split the training set directly and randomly? Such as using the sklearn API:train_test_split？ 
2. Do you  filter out the yes-no answers or comparison questions?
